firestore-counter/functions/src/index.ts

```diff
@@ -16,36 +16,51 @@
 
 import * as functions from "firebase-functions";
 import * as admin from "firebase-admin";
+import { PubSub } from "@google-cloud/pubsub";
 import { ShardedCounterWorker } from "./worker";
 import { ShardedCounterController, ControllerStatus } from "./controller";
 
 admin.initializeApp();
 const firestore = admin.firestore();
 firestore.settings({ timestampsInSnapshots: true });
 
+const pubsub = new PubSub();
+
 const SHARDS_COLLECTION_ID = "_counter_shards_";
 const WORKERS_COLLECTION_ID = "_counter_workers_";
 
 /**
- * Controller is scheduled every minute. It tries to aggregate shards if
+ * The aggregator is scheduled every minute. It tries to aggregate shards if
```

Comment:
Aggregator > Controller

Reply:
I believe that this has been updated to "controllerCore"